src/test/emulators/auth/mfa.spec.ts

```diff
@@ -462,4 +469,210 @@ describeAuthEmulator("mfa enrollment", ({ authApi, getClock }) => {
         expect(res.body.error).to.have.property("message").equals("PROJECT_DISABLED");
       });
   });
+
+  describe("when blocking functions are present", () => {
+    afterEach(async () => {
+      await updateConfig(
+        authApi(),
+        PROJECT_ID,
+        {
+          blockingFunctions: {},
+        },
+        "blockingFunctions"
+      );
+      expect(nock.isDone()).to.be.true;
+      nock.cleanAll();
+    });
+
+    it("mfaSignIn:finalize should update modifiable fields before sign in", async () => {
+      const email = "foo@example.com";
+      const password = "abcdef";
+      const { idToken, localId } = await registerUser(authApi(), { email, password });
+      await updateAccountByLocalId(authApi(), localId, { emailVerified: true });
+      await enrollPhoneMfa(authApi(), idToken, TEST_PHONE_NUMBER);
+
+      getClock().tick(3333);
+
+      const { mfaPendingCredential, mfaEnrollmentId } = await authApi()
+        .post("/identitytoolkit.googleapis.com/v1/accounts:signInWithPassword")
+        .query({ key: "fake-api-key" })
+        .send({ email, password })
+        .then((res) => {
+          expectStatusCode(200, res);
+          expect(res.body).not.to.have.property("idToken");
+          expect(res.body).not.to.have.property("refreshToken");
+          const mfaPendingCredential = res.body.mfaPendingCredential as string;
```

Comment:
nit: extract a helper function for extracting mfaPendingCredential and mfaEnrollmentId from response (mainly to clean the code up visually, since the exact response format is not the focus of the test here)